hack/local-up-cluster.sh

```diff
@@ -475,9 +475,11 @@ function start_kubelet {
       # Docker won't run a container with a cidfile (container id file)
       # unless that file does not already exist; clean up an existing
       # dockerized kubelet that might be running.
+      test_docker
```

Comment:
Cause this [line](https://github.com/xlgao-zju/kubernetes/blob/a79a161c3d40133fb69cad995cada77e8c9a30c8/hack/local-up-cluster.sh#L497) will use docker.